docs/gettinginvolved/infrastructure/documentation.rst

```diff
@@ -0,0 +1,112 @@
+Documentation Hosting
+=====================
+
+Flocker's documentation is hosted on S3, fronted by cloudfront and cloudflare.
+
+S3 Buckets
+----------
+
+There are 3 S3 buckets used for documentation.
+
+clusterhq-docs
+~~~~~~~~~~~~~~
+
+This bucket hosts our public facing documentation.
+
+It has documentation for all marketing releases.
+
+Configuration
+`````````````
+It is configured to allow static website hosting, with an index document of ``index.html``.
+
+It has the following bucket policy configured
+
+.. code:: json
+
+   {
+      "Version": "2008-10-17",
+      "Id": "PolicyForPublicAccess",
+      "Statement": [{
+         "Sid": "1",
+         "Effect": "Allow",
+         "Principal": "*",
+         "Action": "s3:GetObject",
+         "Resource": "arn:aws:s3:::clusterhq-docs/*"
+      }]
+   }
+
+It has logging enabled. (TODO: figure out what configuration we want, or if this should be at a different layer).
```

Comment:
Which configuration is this?  The bucket / path logs are written to?  Or is there more than that?